fastpass/src/main/scala/scala/meta/internal/fastpass/generic/CreateCommand.scala

```diff
@@ -127,35 +126,23 @@ object CreateCommand extends Command[CreateOptions]("create") {
         create.common.workspace.resolve(target.stripPrefix("//"))
       )
 
-    // If we find a recursive wildcard import, then we know it's a bazel import
-    val isBazel = create.bazel || create.targets.exists(tgt =>
-      tgt.endsWith("/...") || tgt.startsWith("//")
-    )
+    // If we find a recursive glob of targets, then we know it's a pants import
+    val isPants =
+      create.forcePants || create.targets.exists(tgt => tgt.endsWith("::"))
 
-    if (isBazel) {
+    if (isPants) {
       val targets = create.targets.map { target =>
-        // Don't touch targets that look like Bazel queries
-        if (!Bazel.isPlainSpec(target)) {
-          target
-        }
-        // Translate foo/bar to foo/bar/... This is inconsistent with Pants (foo/bar:bar),
-        // but more likely to be what the user meant: import directory recursively.
-        else if (
-          !target.contains(":") && !target.endsWith("/...") && isDir(target)
-        ) {
-          target + "/..."
-        }
```

Comment:
I think this part is not covered anymore. Maybe it's for the best actually? 🤔

Reply:
Right now if we get target like foo/bar/... and it's a directory it's translated into foo/bar:: and this case take the same type of target so two cases from pants were translated into one case of bazel so in opposite direction we would have to choose which we want and I chose recursive just to be sure we get all targets.

Reply:
Resolved in [PR-130](https://github.com/scalameta/fastpass/pull/130)